### PR TITLE
add Sturdy core TVL

### PR DESCRIPTION
**Pull request by atozICT20**
##### Twitter Link:
https://twitter.com/SturdyFinance

##### List of audit links if any:
https://www.certik.com/projects/sturdy

##### Website Link:
https://app.sturdy.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):

Logomark_1080_Navy
Logomark_1080_Eggshell

##### Current TVL:
3,684,042.13

##### Chain:
Fantom

##### Short Description (to be shown on DefiLlama):
Sturdy is a first of its kind DeFi protocol for interest-free borrowing and high yield lending. Rather than charging borrowers interest, Sturdy stakes their collateral and passes the yield to lenders. This model changes the relationship between borrowers and lenders to make Sturdy the first positive-sum lending protocol.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Lending

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
TWAP

##### methodology (what is being counted as tvl, how is tvl being calculated):
Counts deposited tokens.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/sturdy/index.js exports TVL: 
                
 
 
fantom                    3.68 M
fantom-borrow             34.77 k
borrow                    34.77 k

total                    3.68 M

File: projects/sturdy/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+`;
+
+const priceInUSD = (value, decimals, price) => {
+    return (value / Math.pow(10, decimals) * price / Math.pow(10, 8)).toFixed(2);
+}
+
+async function fetch(borrow) {
+    const { reserves } = await request(graphUrl, graphQuery);
+    const tvl = reserves.reduce((sum, reserve) => {
+        const value = borrow ? reserve.totalLiquidity - reserve.availableLiquidity : reserve.totalDeposits;
```
Comment (waynebruce0x) [inline projects/sturdy/index.js:30]: can this be changed to ... : reserves.availableLiquidity;

Comment (atozICT20) [reply projects/sturdy/index.js:30]: this should be `reserve.totalDeposits;`

Comment (waynebruce0x) [reply projects/sturdy/index.js:30]: we exclude borrows from core tvl, so it should be supply - borrows. Users can filter on the borrows on our UI

Comment (atozICT20) [reply projects/sturdy/index.js]: Thank you for your guidance.
I changed the calculation logic and pushed it. 
Please review once more.

**Pull request merged**